### PR TITLE
Docker Development Setup

### PR DESCRIPTION
**Pull request by arnoldokoth**
What does this PR do?
* Automate development environment setup locally using Docker

How can this be manually tested? 
* Follow the process documented in the README file under the `Local Development Docker Setup` section. 

Reach out to @daumie or myself in case of any challenges during the setup and anything that might need clarification.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,4 +60,37 @@ This repository contains the API endpoints and models for the ART project implem
...
+### Start The Services
+To start the services you run the make start task i.e.
+```
+$ make start
+```
+
+### Create the superuser account
+To create the superuser account, we are going to first connect the container of our application using the exec task in the Makefile. Run the task as shown:
+```
+$ make exec
```
Comment (egichuri) [inline README.md]: The terminal session exits after running one command.. any way we can get around that?

Comment (arnoldokoth) [reply README.md]: We are yet to find one. We were also wondering what was causing that. 🤔

Comment (egichuri) [reply README.md]: Cool.. we can go with this for now

**Comment by egichuri** [comment]
@arnoldokoth @daumie I think we can merge this

**Pull request merged**